### PR TITLE
Add Support for Symfony5

### PR DESCRIPTION
**Pull request by rmanibus**
dependency-injection-extra version (dev-symfony5) must be replaced with the new release regarding https://github.com/mariusbalcytis/dependency-injection-extra/pull/5

### CODE REVIEW

**Comment by rmanibus** [comment]
Backward compatibility:
FilterResponseEvent and GetResponseEvent deprecated in Symfony 4.3, deleted in Symfony 5.0

File: composer.json [hunk 1]
```diff
@@ -3,25 +3,32 @@
...
+        "url": "https://github.com/rmanibus/dependency-injection-extra"
+      }
+    ],
     "require": {
         "php": "^7.0",
         "maba/gentle-force": "^0.2.1|^0.3",
-        "symfony/framework-bundle": "^3.4.26|^4.2.7",
-        "maba/dependency-injection-extra": "^0.1.1|^1.0",
+        "symfony/framework-bundle": "^4.3|^5.0",
+        "maba/dependency-injection-extra": "dev-symfony5|^0.1.1|^1.0",
```
Comment (mariusbalcytis) [inline composer.json]: Already has support - we should drop vustom repository and custom constraint

File: tests/Functional/Fixtures/config/common_basic.yml [hunk 2]
```diff
@@ -3,5 +3,8 @@ imports:
 
 services:
     logger:
-        class: Symfony\Component\Debug\BufferingLogger
+        class: Symfony\Component\ErrorHandler\BufferingLogger
         public: true
+        #class: Symfony\Component\HttpKernel\Log\Logger
```
Comment (mariusbalcytis) [inline tests/Functional/Fixtures/config/common_basic.yml]: Could we remove commented code?

Comment (rmanibus) [reply tests/Functional/Fixtures/config/common_basic.yml]: yes I was debugging the tests, just forgot to remove it !

**Comment by mariusbalcytis** [comment]
Could you also drop 7.0 from travis configuration? As there are no more compatible symfony versions.

And could you see the php-cs-fixer, that also fails?

**Comment by rmanibus** [comment]
I think it's ok now

**Pull request merged**